velox/exec/AggregateWindow.cpp

```diff
@@ -391,6 +391,7 @@ void registerAggregateWindowFunction(const std::string& name) {
         [name](
             const std::vector<exec::WindowFunctionArg>& args,
             const TypePtr& resultType,
+            bool /*ignoreNulls*/,
```

Comment:
Does 'ignoreNulls' not apply to aggregate windows? Perhaps, mentions this in the PR description.